waterdata/location.py

```diff
@@ -0,0 +1,173 @@
+"""
+Utility functions and classes for working with
+USGS water services.
+
+"""
+from collections import namedtuple
+import datetime
+
+from .utils import execute_get_request, parse_rdb
+
+from . import app
+
+
+Parameter = namedtuple('Parameter', ['parameter_cd', 'start_date', 'end_date', 'record_count'])
+
+
+class MonitoringLocation:
+
+    service_endpoint = app.config['SERVICE_ROOT']
+
+    def __init__(self, monitoring_location_record):
+        """
+        A monitoring location object.
+
+        Attributes:
+            location_number (str): identifier for the monitoring location
```

Comment:
I decided to retain as attribute names as I get the sense that we don't want to call these places sites in the future.

Reply:
Your sense is correct